python/cudf/cudf/_lib/text.pyx

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
+
+import cudf
+
+from cython.operator cimport dereference
+from libcpp.memory cimport make_unique, unique_ptr
+from libcpp.string cimport string
+from libcpp.utility cimport move
+
+from cudf._lib.column cimport Column
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.io.text cimport (
+    data_chunk_source,
+    make_source,
+    make_source_from_file,
+    multibyte_split,
+)
+
+
+def read_text(object filepaths_or_buffers,
+              object delimiter=None):
+    """
+    Cython function to call into libcudf API, see `read_text`.
```

Comment:
Is the "see `read_text`" referring to `cudf.io.read_path`?

Reply:
No, sorry the name had changed on the C++ side since I made this. It is actually `multibyte_split` at https://github.com/rapidsai/cudf/blob/branch-21.10/cpp/include/cudf/io/text/multibyte_split.hpp#L30 I will update that